### PR TITLE
Always send a `TaskNotificationType` with a 2XX HTTP status code

### PR DESCRIPTION
**Pull request by jbern0rd**

### CODE REVIEW

**Review by mcornaton — APPROVED** [review]

File: src/main/java/com/iexec/core/replicate/ReplicatesController.java [hunk 1]
```diff
@@ -21,6 +21,7 @@
 import com.iexec.common.replicate.*;
 import com.iexec.core.security.JwtTokenProvider;
 import com.iexec.core.worker.WorkerService;
+import feign.FeignException;
```
Comment (jeremyjams) [inline src/main/java/com/iexec/core/replicate/ReplicatesController.java:24]:
```suggestion
```

**Review by jeremyjams — APPROVED** [review]
OK, thanks :+1:

File: src/main/java/com/iexec/core/replicate/ReplicatesController.java (hunk 1 above)
Comment (jbern0rd) [reply src/main/java/com/iexec/core/replicate/ReplicatesController.java:24]: The dependency required after  @mcornaton recommended me to use a link in javadoc comment.
I could remove it by replacing {@link} with {@code} but the integration is not the same in IDE.

Comment (jeremyjams) [reply src/main/java/com/iexec/core/replicate/ReplicatesController.java:24]: Interesting :bulb:

**Pull request merged**